olm-catalog/serverless-operator/Dockerfile

```diff
@@ -7,8 +7,8 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=serverless-operator
-LABEL operators.operatorframework.io.bundle.channel.default.v1="4.3"
-LABEL operators.operatorframework.io.bundle.channels.v1="4.5,4.6"
+LABEL operators.operatorframework.io.bundle.channel.default.v1="4.6"
+LABEL operators.operatorframework.io.bundle.channels.v1="4.6"
```

Comment:
glad to see this defaulting

Reply:
Yeah, me too. And I'm sure @cardil as well. :)